core/market/decentralized/src/matcher/matcher.rs

```diff
@@ -287,20 +297,20 @@ async fn on_offer_unsubscribed(db: DbExecutor, msg: OfferUnsubscribed) -> Result
             .as_dao::<OfferDao>()
             .remove_offer(&msg.subscription_id)
             .await
-            .map_err(|_| {
+            .map_err(|error| {
                 log::warn!(
                     "Failed to remove offer [{}] during unsubscribe.",
                     &msg.subscription_id
                 );
             });
-        Ok(Propagate::True)
+        Result::<_, UnsubscribeError>::Ok(Propagate::True)
     }
     .await
-    .or_else(|e| {
-        let reason = match e {
+    .or_else(|error| {
+        let reason = match error {
             UnsubscribeError::OfferExpired => StopPropagateReason::Expired,
             UnsubscribeError::AlreadyUnsubscribed => StopPropagateReason::AlreadyUnsubscribed,
-            _ => StopPropagateReason::Error(e.to_string()),
+            _ => StopPropagateReason::Error(format!("{}", error)),
```

Comment:
isn't the `e.to_string()` more readable to you?

https://doc.rust-lang.org/std/fmt/#usage
> To convert a single value to a string, use the to_string method.